src/index.test.ts

```diff
@@ -3023,29 +3023,71 @@ describe('concatSig', function () {
   });
 });
 
-it('normalize address lower cases', function () {
-  const initial = '0xA06599BD35921CfB5B71B4BE3869740385b0B306';
-  const result = sigUtil.normalize(initial);
-  expect(result).toBe(initial.toLowerCase());
-});
+describe('normalize', function () {
+  it('should normalize an address to lower case', function () {
+    const initial = '0xA06599BD35921CfB5B71B4BE3869740385b0B306';
+    const result = sigUtil.normalize(initial);
+    expect(result).toBe(initial.toLowerCase());
+  });
 
-it('normalize address adds hex prefix', function () {
-  const initial = 'A06599BD35921CfB5B71B4BE3869740385b0B306';
-  const result = sigUtil.normalize(initial);
-  expect(result).toBe(`0x${initial.toLowerCase()}`);
-});
+  it('should normalize address without a 0x prefix', function () {
+    const initial = 'A06599BD35921CfB5B71B4BE3869740385b0B306';
+    const result = sigUtil.normalize(initial);
+    expect(result).toBe(`0x${initial.toLowerCase()}`);
+  });
 
-it('normalize an integer converts to byte-pair hex', function () {
-  const initial = 1;
-  const result = sigUtil.normalize(initial);
-  expect(result).toBe('0x01');
-});
+  it('should normalize an integer to a byte-pair hex string', function () {
+    const initial = 1;
+    const result = sigUtil.normalize(initial);
+    expect(result).toBe('0x01');
+  });
 
-it('normalize an unsupported type throws', function () {
-  const initial = {};
-  expect(() => sigUtil.normalize(initial as any)).toThrow(
-    'eth-sig-util.normalize() requires hex string or integer input. received object:',
-  );
+  // TODO: Add validation to disallow negative integers.
+  it('should normalize a negative integer to 0x', function () {
+    const initial = -1;
+    const result = sigUtil.normalize(initial);
+    expect(result).toBe('0x');
+  });
+
+  // TODO: Add validation to disallow null.
+  it('should return undefined if given null', function () {
+    const initial = null;
+    expect(sigUtil.normalize(initial as any)).toBeUndefined();
+  });
+
+  // TODO: Add validation to disallow undefined.
+  it('should return undefined if given undefined', function () {
+    const initial = undefined;
+    expect(sigUtil.normalize(initial as any)).toBeUndefined();
+  });
+
+  it('should throw if given an object', function () {
+    const initial = {};
+    expect(() => sigUtil.normalize(initial as any)).toThrow(
+      'eth-sig-util.normalize() requires hex string or integer input. received object:',
+    );
+  });
+
+  it('should throw if given a boolean', function () {
+    const initial = true;
+    expect(() => sigUtil.normalize(initial as any)).toThrow(
+      'eth-sig-util.normalize() requires hex string or integer input. received boolean: true',
+    );
+  });
+
+  it('should throw if given a BigInt', function () {
```

Comment:
The primitive type is `bigint` apparently
```suggestion
  it('should throw if given a bigint', function () {
```